embedding/cpp/include/flutter_app.h

```diff
@@ -15,10 +15,22 @@
 
 #include "flutter_engine.h"
 
+enum class FlutterRendererType {
+  // The renderer based on EvasGL.
+  kEvasGL,
+  // The renderer based on EGL.
+  kEGL,
+};
+
 // The app base class for headed Flutter execution.
 class FlutterApp : public flutter::PluginRegistry {
  public:
-  explicit FlutterApp() {}
+  explicit FlutterApp()
+#ifdef WEARABLE_PROFILE
+      : renderer_type_(FlutterRendererType::kEvasGL)
+#endif
+  {
+  }
```

Comment:
nit: This might look cleaner. (Would there be any logical difference?)
```suggestion
  explicit FlutterApp() {
#ifdef WEARABLE_PROFILE
    renderer_type_ = FlutterRendererType::kEvasGL;
#endif
  }
```

Reply:
```cpp
A a = 1;
```
```cpp
A a;
a = 1;
```
I know the difference between these.

Reply:
I think it doesn't matter now. I like looks cleaner too